src/components/frame/RouteLayouts/UserNavigation/Wallet/Wallet.tsx

```diff
@@ -32,8 +32,8 @@ const MSG = defineMessages({
 });
 
 const Wallet = () => {
-  const [walletConnecting, setWalletConnecting] = useState<boolean>(false);
```

Comment:
Moving this to `appContext` makes it accessible elsewhere, for example in `UserProfile`. Here it's useful because the text rendered in the event there are no colonies is dependent on which wallet is connected, and so we can show a loading state while the wallet is connecting.